browser_history/generic.py

```diff
@@ -128,15 +128,17 @@ def __init__(self, plat: typing.Optional[utils.Platform] = None):
             plat = utils.get_platform()
         homedir = Path.home()
 
-        error_string = self.name + " browser is not supported on {}"
+        error_string = (
+            self.name + f" browser is not supported on {utils.get_platform_name(plat)}"
+        )
```

Comment:
You should maybe remove the concatenation and just use f string.
```python
f"{self.name} browser is not supported on {utils.get_platform_name(plat)}"
```

Reply:
Good point, I didn't notice.

Reply:
All in all LGTM